src/components/Detail/_detail.scss

```diff
@@ -0,0 +1,22 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-detail {
+  display: flex;
+  justify-content: space-between;
```

Comment:
@aahill50 Rather than a `space-between`, I see a `gap: $spacing-md` in the Figma. 
![image](https://github.com/user-attachments/assets/71610251-c8f8-4fca-86dc-35966cf668e5)
There's also some tricky bits going on where they all align with each other.  Can you create a `DetailList` component that will make sure they all line up on the longest `Detail` and make a story to match this Figma content: https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=11364-24078&m=dev so we make sure it works correctly?

Reply:
@scottdickerson I updated the component, added a DetailsContainer and added some props that determine how the Details are laid out in the container. It supports both the justified layout seen in the Estimate/Starting Bid stack, and the column type layout seen in the Lot Details stack. I also added a prop for showing or hiding the separators since that also differs between the 2 use cases